addons/stock/models/stock_move.py

```diff
@@ -937,6 +937,7 @@ def remove_prefix(text, prefix):
             lot_names = self.env['stock.lot'].generate_lot_names(first_lot, len(lot_qties))
         elif mode == 'import':
             lot_names = self.split_lots(lot_text)
+            lot_qties = [1] * len(lot_names)
```

Comment:
in the case of quantity 50 and qty/lot 6 generate_lot_names should generate 9 lots even if the count is 8 (the correct value is computed in "generate_lot_qty")
it's better to keep it as is to avoid duplication

ref: test_generate_serial_2